content/neuvector-prime/azure/contents.lr

```diff
@@ -104,6 +104,53 @@ The Neuvector login will appear, login with the credential which you have set du
 
 Once logged in, you can begin to [navigate and configure NeuVector](https://open-docs.neuvector.com/navigation/navigation).
 
+
+
+```
+Important! To change the Admin password after initial login. Edit profile in the 
```

Comment:
Why is this important if we set it in the UI when deploying. we probably need to explain why.  i.e.  The current implementation exposes the original password in the deployment screen (Screenshot).  Until this is resolved, it is suggested to change the Admin password using the Neuvector console (Screenshot).  Then add the note about not touching the values in settings

Reply:
Done